bug:  go version in the `scripts/docker/kclvm-builder/Dockerfile` is too low and module requires Go 1.19

## Bug Report

I'm trying to fix #396 , but upon investigation came across this bug.

### 1. Minimal reproduce step (Required)

`cd scripts/docker/kclvm-builder && make kclvm-builder`

### 2. What did you expect to see? (Required)

This docker image is successfully built.

### 3. What did you see instead (Required)

I deleted the irrelevant Dockerfile configuration and the test results are as follows:

```shell
Step 9/15 : RUN go get honnef.co/go/tools/cmd/...
 ---> Running in e0be377fb8a3
go: downloading honnef.co/go/tools v0.4.3
go: downloading golang.org/x/tools v0.4.1-0.20221208213631-3f74d914ae6d
go: downloading golang.org/x/exp/typeparams v0.0.0-20221208152030-732eee02a75a
go: downloading github.com/BurntSushi/toml v1.2.1
go: downloading golang.org/x/sys v0.3.0
go: downloading golang.org/x/mod v0.7.0
go get: installing executables with 'go get' in module mode is deprecated.
        Use 'go install pkg@version' instead.
        For more information, see https://golang.org/doc/go-get-install-deprecation
        or run 'go help get' or 'go help install'.
# honnef.co/go/tools/go/ast/astutil
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/ast/astutil/util.go:133:8: undefined: ast.IndexListExpr
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/ast/astutil/util.go:136:12: cannot assign ast.Expr to cp.X in multiple assignment
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/ast/astutil/util.go:141:22: cannot assign ast.Expr to cp.Indices[i] in multiple assignment
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/ast/astutil/util.go:281:8: undefined: ast.IndexListExpr
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/ast/astutil/util.go:282:12: undefined: ast.IndexListExpr
note: module requires Go 1.19
# honnef.co/go/tools/go/types/typeutil
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/types/typeutil/upstream.go:26:12: syntax error: unexpected any, expecting ]
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/types/typeutil/upstream.go:30:13: syntax error: unexpected [, expecting comma or )
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/types/typeutil/upstream.go:31:13: syntax error: unexpected [, expecting comma or )
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/types/typeutil/upstream.go:35:3: syntax error: non-declaration statement outside function body
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/types/typeutil/upstream.go:40:13: syntax error: unexpected [, expecting comma or )
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/types/typeutil/upstream.go:41:13: syntax error: unexpected [, expecting comma or )
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/types/typeutil/upstream.go:42:13: syntax error: unexpected [, expecting comma or )
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/types/typeutil/upstream.go:43:48: method has multiple receivers
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/types/typeutil/upstream.go:43:48: syntax error: unexpected {, expecting name or (
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/types/typeutil/upstream.go:49:13: syntax error: unexpected [, expecting comma or )
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/types/typeutil/upstream.go:49:13: too many errors
note: module requires Go 1.19
# honnef.co/go/tools/pattern
go/pkg/mod/honnef.co/go/tools@v0.4.3/pattern/convert.go:16:35: undefined: ast.IndexListExpr
go/pkg/mod/honnef.co/go/tools@v0.4.3/pattern/convert.go:138:9: assignment mismatch: 2 variables but 1 value
go/pkg/mod/honnef.co/go/tools@v0.4.3/pattern/match.go:88:25: undefined: any
note: module requires Go 1.19
# honnef.co/go/tools/knowledge
go/pkg/mod/honnef.co/go/tools@v0.4.3/knowledge/signatures.go:9:22: undefined: types.NewSignatureType
go/pkg/mod/honnef.co/go/tools@v0.4.3/knowledge/signatures.go:21:23: undefined: types.NewSignatureType
go/pkg/mod/honnef.co/go/tools@v0.4.3/knowledge/signatures.go:32:35: undefined: types.NewSignatureType
go/pkg/mod/honnef.co/go/tools@v0.4.3/knowledge/signatures.go:43:42: undefined: types.NewSignatureType
go/pkg/mod/honnef.co/go/tools@v0.4.3/knowledge/signatures.go:52:43: undefined: types.NewSignatureType
go/pkg/mod/honnef.co/go/tools@v0.4.3/knowledge/signatures.go:61:27: undefined: types.NewSignatureType
note: module requires Go 1.19
# honnef.co/go/tools/go/loader
go/pkg/mod/honnef.co/go/tools@v0.4.3/go/loader/loader.go:233:4: unknown field 'Instances' in struct literal of type types.Info
note: module requires Go 1.19
```

### 4. What is your KusionStack components version? (Required)

<!-- Paste the output of Kusion/kclvm-go/KCLVM version -->

--- a/scripts/docker/kclvm-builder-centos8/Dockerfile
+++ b/scripts/docker/kclvm-builder-centos8/Dockerfile
@@ -41,24 +41,30 @@ RUN yum -y install llvm-devel
 RUN yum -y install libffi-devel
 RUN ln -s /usr/lib64/libtinfo.so.6 /usr/lib64/libtinfo.so
 
-# Go 1.6
-RUN yum -y install golang
-RUN go version
+# golang 1.19+
+RUN mkdir -p /root/download && cd /root/download \
+    && wget https://dl.google.com/go/go1.20.5.linux-amd64.tar.gz \
+    && tar -zxvf go1.20.5.linux-amd64.tar.gz \
+    && mv ./go /usr/local/go1.20.5
+RUN ln -sf /usr/local/go1.20.5/bin/go /usr/bin/go
+RUN rm -rf /root/download
+
+ENV GOPATH=/go
+ENV GOLANG_VERSION=1.20.5
+
+RUN go install golang.org/x/lint/golint@latest
+RUN go install golang.org/x/tools/cmd/goimports@latest
+# RUN go install honnef.co/go/tools/cmd/...@latest
+
+RUN go install github.com/t-yuki/gocover-cobertura@latest
+RUN go install github.com/jstemmer/go-junit-report@latest
+
+RUN rm -rf /go/pkg/mod
+RUN rm -rf /go/pkg/sumdb
 
 # /usr/lib64/python3.9
 RUN yum -y install python39-devel
 RUN python3 -m pip install pytest
-
-# golang apps
-RUN go get golang.org/x/lint/golint
-RUN go get golang.org/x/tools/cmd/goimports
-#RUN go get honnef.co/go/tools/cmd/...
-
-RUN go get github.com/t-yuki/gocover-cobertura
-RUN go get github.com/jstemmer/go-junit-report
-
-RUN rm -rf /go/pkg/mod
-RUN rm -rf /go/pkg/sumdb
 
 RUN ln -sf /usr/share/zoneinfo/Asia/Shanghai /etc/localtime
 RUN echo 'Asia/Shanghai' >/etc/timezone

--- a/scripts/docker/kclvm-builder-centos8/Makefile
+++ b/scripts/docker/kclvm-builder-centos8/Makefile
@@ -26,6 +26,6 @@ publish-builder:
 	@echo "push ${BUILDER_IMAGE} ok"
 
 sh-in-builder:
-	${RUN_IN_DOCKER} -v ${PWD}/../..:/root/kclvm -w /root ${BUILDER_IMAGE} bash
+	${RUN_IN_DOCKER} -v ${PWD}/../../..:/root/kclvm -w /root ${BUILDER_IMAGE} bash
 
 clean:

--- a/scripts/docker/kclvm-builder/Dockerfile
+++ b/scripts/docker/kclvm-builder/Dockerfile
@@ -43,23 +43,23 @@ RUN apt-get install -y clang-12 lld-12
 RUN ln -sf /usr/bin/clang-12   /usr/bin/clang
 RUN ln -sf /usr/bin/wasm-ld-12 /usr/bin/wasm-ld
 
-# golang 1.17+
+# golang 1.19+
 RUN mkdir -p /root/download && cd /root/download \
-  && wget https://dl.google.com/go/go1.17.3.linux-amd64.tar.gz \
-  && tar -zxvf go1.17.3.linux-amd64.tar.gz \
-  && mv ./go /usr/local/go1.17.3
-RUN ln -sf /usr/local/go1.17.3/bin/go /usr/bin/go
+  && wget https://dl.google.com/go/go1.20.5.linux-amd64.tar.gz \
+  && tar -zxvf go1.20.5.linux-amd64.tar.gz \
+  && mv ./go /usr/local/go1.20.5
+RUN ln -sf /usr/local/go1.20.5/bin/go /usr/bin/go
 RUN rm -rf /root/download
 
 ENV GOPATH=/go
-ENV GOLANG_VERSION=1.17.3
+ENV GOLANG_VERSION=1.20.5
 
-RUN go get golang.org/x/lint/golint
-RUN go get golang.org/x/tools/cmd/goimports
-RUN go get honnef.co/go/tools/cmd/...
+RUN go install golang.org/x/lint/golint@latest
+RUN go install golang.org/x/tools/cmd/goimports@latest
+RUN go install honnef.co/go/tools/cmd/...@latest
 
-RUN go get github.com/t-yuki/gocover-cobertura
-RUN go get github.com/jstemmer/go-junit-report
+RUN go install github.com/t-yuki/gocover-cobertura@latest
+RUN go install github.com/jstemmer/go-junit-report@latest
 
 RUN rm -rf /go/pkg/mod
 RUN rm -rf /go/pkg/sumdb

